cpp/src/io/parquet/writer_impl.hpp

```diff
@@ -234,6 +262,19 @@ class writer::impl {
   Compression compression_           = Compression::UNCOMPRESSED;
   statistics_freq stats_granularity_ = statistics_freq::STATISTICS_NONE;
   bool int96_timestamps              = false;
+  // Cuda stream to be used
+  rmm::cuda_stream_view stream_ = rmm::cuda_stream_default;
+  // Overall file metadata.  Filled in during the process and written during write_chunked_end()
+  cudf::io::parquet::FileMetaData md;
+  // optional user metadata
+  table_metadata_with_nullability user_metadata_with_nullability;
+  // special parameter only used by detail::write() to indicate that we are guaranteeing
+  // a single table write.  this enables some internal optimizations.
```

Comment:
looks like the two comments got switched around (for user_metadata and single_write_mode).